### PR TITLE
Storing and Retrieving body and headers from filesystem

### PR DESCRIPTION
**Pull request by Hirobreak**
- Storing email body and headers in filesystem when receiving files or in sync
- Retrieving email body from filesystem when displaying email or for sync

### CODE REVIEW

File: electron_app/electron-starter.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ require('./src/ipc/loading.js');
 require('./src/ipc/login.js');
 require('./src/ipc/mailbox.js');
 require('./src/ipc/database.js');
-
+require('./src/ipc/client.js');
```
Comment (JulianAdams4) [inline electron_app/electron-starter.js]: Delete `require('./src/ipc/client.js')`
Itś called within `initApp()`

File: electron_app/src/utils/PrintUtils.js [hunk 2]
```diff
@@ -126,7 +149,7 @@ const printEmailOrThread = async ({ emailId, threadId }) => {
         printDocumentTemplateClose;
     }
   }
-
+  console.log('que tiro weh');
```
Comment (erikaperugachi) [inline electron_app/src/utils/PrintUtils.js]: delete

File: electron_app/src/utils/PrintUtils.js [hunk 3]
```diff
@@ -144,6 +167,7 @@ const printEmailOrThread = async ({ emailId, threadId }) => {
         defaultDocumentName
       );
     });
+    console.log('que tiro weh 2');
```
Comment (erikaperugachi) [inline electron_app/src/utils/PrintUtils.js]: delete

File: email_composer/src/utils/ipc.js [hunk 4]
```diff
@@ -44,6 +44,17 @@ export const postEmail = async params => {
   return await callMain('client-post-email', params);
 };
 
+/* File System
+   ----------------------------- */
+
```
Comment (erikaperugachi) [inline email_composer/src/utils/ipc.js]: delete enter

File: email_mailbox/src/utils/ipc.js [hunk 5]
```diff
@@ -68,6 +68,17 @@ export const sendPrintThreadEvent = threadId => {
   callMain('print-to-pdf', { threadId });
 };
 
+/* File System
+   ----------------------------- */
+
```
Comment (erikaperugachi) [inline email_mailbox/src/utils/ipc.js]: delete enter

File: electron_app/src/dbExporter.js [hunk 6]
```diff
@@ -427,6 +454,20 @@ const importDatabaseFromFile = async ({ filepath, databasePath }) => {
   });
 };
 
+const storeEmailBodies = emailRows => {
+  const username = `${myAccount.recipientId}@${APP_DOMAIN}`;
+  console.log(username);
```
Comment (erikaperugachi) [inline electron_app/src/dbExporter.js]: delete

**Pull request merged**